pkg/task/task.go

```diff
@@ -1578,8 +1584,10 @@ func (m *executor) listOEV1alpha1URRaw(opts metav1.ListOptions) (result []byte,
 		WithNamespace(m.getTaskRunNamespace()).
 		List(opts)
 	if err != nil {
+		err = errors.Wrap(err, "failed to list upgraderesult")
 		return
 	}
 	result, err = json.Marshal(urList)
+	err = errors.Wrap(err, "failed to list upgraderesult")
```

Comment:
check for err & then wrap.
make sure to push urList.String() if stringer is implemented or %+v otherwise